### PR TITLE
Feature: working directory

### PR DESCRIPTION
**Pull request by artemii-yanushevskyi**

### CODE REVIEW

File: content/getting-started/yaml.md [hunk 1]
```diff
@@ -401,3 +401,30 @@ workflows:
             excludes:
               - '**/*.md'
 ```
+
+## Working directory
+
+You may select working directory globally for the entire workflow or for locally individual scripts. If not specified, global working directory defaults to the directory where the repository is cloned (`/Users/builder/clone`). You can override global working directory by specifying working directory in the individual steps. Consider the example below:
```
Comment (helinanever) [inline content/getting-started/yaml.md:407]:
```suggestion
You may select a working directory globally for the entire workflow or for individual scripts only. If not specified, the global working directory defaults to the directory where the repository is cloned (`/Users/builder/clone`). You can override the global working directory by specifying the working directory in the individual steps. Consider the example below:
```

File: content/getting-started/yaml.md [hunk 2]
```diff
@@ -401,3 +401,30 @@ workflows:
...
+        working_directory: mobile/ios
+      - name: Build Android
+        script: # Android build command
+        working_directory: mobile/android
+      - name: Logs
+        script: # Process logs in /Users/builder/Library/Logs folder
+        working_directory: /Users/builder/Library/Logs
+```
+
+Relative path to working directory are counted based on repository clone directory, e.g. if `mobile` is the working directory then the script will be executed in `/Users/builder/clone/mobile`.
```
Comment (helinanever) [inline content/getting-started/yaml.md:428]:
```suggestion
Working directory paths are relative to the repository clone directory, e.g. if `mobile` is the working directory, then the script will be executed in `/Users/builder/clone/mobile`.
```

File: content/getting-started/yaml.md [hunk 3]
```diff
@@ -401,3 +401,30 @@ workflows:
...
+workflows:
+  build-ios:
+    name: Build iOS and Android
+    working_directory: mobile/
+    scripts:
+      - name: Prepare
+        script: # current working directory is /Users/builder/clone/mobile
+      - name: Build iOS
+        script: # iOS build command
+        working_directory: mobile/ios
```
Comment (helinanever) [inline content/getting-started/yaml.md:419]: should all working directory paths end with a slash for consistency? compare: mobile/ and mobile/ios

File: content/getting-started/yaml.md [hunk 4]
```diff
@@ -401,3 +401,30 @@ workflows:
...
+        script: # Android build command
+        working_directory: mobile/android
+      - name: Logs
+        script: # Process logs in /Users/builder/Library/Logs folder
+        working_directory: /Users/builder/Library/Logs
+```
+
+Relative path to working directory are counted based on repository clone directory, e.g. if `mobile` is the working directory then the script will be executed in `/Users/builder/clone/mobile`.
+
+Note that you could specify absolute path as a working directory as well.
```
Comment (helinanever) [inline content/getting-started/yaml.md:430]:
```suggestion
Note that you can specify an absolute path as a working directory as well.
```

File: content/getting-started/yaml.md (hunk 3 above)
Comment (artemii-yanushevskyi) [reply content/getting-started/yaml.md:419]: not necessarily, slash is optional

Comment (artemii-yanushevskyi) [reply content/getting-started/yaml.md:419]: I removed slash from `mobile/`

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**